src/main/java/no/nav/pto/veilarbportefolje/aktiviteter/AktivitetService.java

```diff
@@ -45,11 +45,12 @@ public void behandleKafkaMeldingLogikk(KafkaAktivitetMelding aktivitetData) {
         AktorId aktorId = AktorId.of(aktivitetData.getAktorId());
         boolean bleProsessert = aktivitetDAO.tryLagreAktivitetData(aktivitetData);
 
-        if (bleProsessert && (aktivitetData.isAvtalt() || brukIkkeAvtalteAktiviteter(unleashService))) {
-            utledAktivitetstatuserForAktoerid(aktorId);
-            elasticIndexer.indekser(aktorId);
+        if (aktivitetData.getVersion() > 49179897) {
```

Comment:
why this number 49179897?

Reply:
That was the latest version that we have read in prod